src/main/java/mate/academy/internetshop/dao/jdbc/BucketDaoJdbcImpl.java

```diff
@@ -97,7 +97,6 @@ public Bucket deleteById(Long id) {
         return deletedBucket;
     }
 
-    @Override
```

Comment:
@kozhukhovsky why you have removed this?

Reply:
because i removed this methods from interface